openfl/federated/task/runner_keras.py

```diff
@@ -17,8 +18,8 @@
 
 with catch_warnings():
     simplefilter(action="ignore")
+    import keras as ke
```

Comment:
I realize this is carried over from prior examples, but `ke` is not a standard abbreviation for `keras`: https://keras.io/examples/vision/image_classification_with_vision_transformer/

While all of the keras code is being updated, I would recommend changing this to the following to make the code slightly easier to read for developers:

```suggestion
    import keras
```

Reply:
done